specification/paths/traffic/trafficRoute.yml

```diff
@@ -0,0 +1,42 @@
+servers:
+  - url: https://api.woosmap.com
+tags:
+  - Traffic API
+description: >
+  Get distance, duration and path (as a polyline) for a pair of origin and destination, based on the recommended route between those two points (including traffic).
```

Comment:
Get distance, duration and path (as a polyline) for a pair of origin and destination, based on the recommended route between those two points, while taking traffic into account.